lib/nimble_template/templates/variants/phoenix/template.ex

```diff
@@ -67,6 +67,9 @@ defmodule NimbleTemplate.Templates.Phoenix.Template do
     if install_addon_prompt?("Oban"), do: PhoenixAddons.Oban.apply(project)
     if install_addon_prompt?("ExVCR"), do: PhoenixAddons.ExVCR.apply(project)
 
+    if install_addon_prompt?("Gettext Extract and Merge command"),
```

Comment:
should we name it as `Gettext Helpers` @byhbt 🤔 

`if install_addon_prompt?("Gettext Helpers"),`

Reply:
yup, as [discussed](https://github.com/nimblehq/elixir-templates/pull/138#discussion_r809772565) with Olivier, we agreed that we can make this addon as default because initially the generated Phoenix project already included the `Gettext`, so I will remove this `prompt` also.